### ISSUE TITLE
auth fb problem

### ISSUE THREAD

**Issue by diego04** [hapijs/bell#145]
in your example in the main page you have the path 'login':
I haven't tried other providers. I've only been using facebook. 
If you do multiple request per second in '/login'
you can even doing it by opening a new tab in the browser real fast and pasting 'localhost:8000/login' consecutively. eventually, you will get a '500 error', what should we do here?

**Comment by ldesplat** [comment]
Do you know if the `500` error is from us or because Facebook returned with it. I would assume that if you keep spamming the same user that Facebook would rate limit you?

We may have some sort of issue but this seems like something Facebook would protect against?

**Comment by diego04** [comment]
Yes, I think (have not tested) that it is because of facebook returning null or error. Im not too sure if the 500 is from facebook or bell. In any case, what do you recommend for us  (users of bell) to do in this situation. Should we just do a try catch and where is the best place to put it?

**Comment by ldesplat** [comment]
If you could reproduce this once more but this time gather more information, you can copy/paste some of the information here and I can help to diagnose it.

The `500` error is obviously not giving you any information but you can use [hapijs/good](https://github.com/hapijs/good) and log the error events and we should have more information as to where the `500` comes from.

You can also use [hapijs/tv](https://github.com/hapijs/tv) for debugging. I really like it for tracing things :)

**Comment by diego04** [comment]
Alright, I'll get back to you

**Comment by ldesplat** [comment]
@diego04 Do you have any updates on this? Do you still think it's potentially a Bell issue?

**Comment by diego04** [comment]
Im trying out good and tv seems to be broken. I've just started learning hapijs/good the basic configuration  doesn't give me enough information. A quick fix I did was to prevent access that url multiple times. I believe it is mostly an issue with fb response, in normal situations bell works great. I'll let you know more if I can figure out how to properly configure hapijs/good to give me a more detailed log.
